app/krs.js

```diff
@@ -124,6 +124,24 @@ exports.provisionKey = co(function *(req) {
     throw utils.ErrorResponse(400, 'email required');
   }
 
+  key.custom = req.body.custom || {};
+  key.custom.created = new Date();
+
+  /**
+   * Here we handle the krsSpecific parameter
+   * This parameter should be an object containing custom data relevant to your KRS policies
+   * This is an optional parameter
+   */
+  const krsSpecific = req.body.krsSpecific;
+  if (krsSpecific) {
```

Comment:
Do we need this hook at all? Is it possible to pass any custom data the (key recovery) service provider requires in the `custom` body param? 

Isn't that what `custom` is used for? 

.....

I guess I'm just having trouble understanding if this is something specific that one service provider requires vs something that is generic enough that multiple service providers would use.

If it's the former, I would recommend making NO change and having them fork the project to add this themselves --- that's essentially what they'll be doing anyway right? Seems like the expectation is that they'll pull the project down, make some local changes (that will never be checked in nor pushed upstream) and running the project. Unless we _always_ want to execute custom, service provider specific code in this one explicit spot, then I'm skeptical that we need to introduce this change at all.

I have much less context though -- maybe @dacruz21 has a stronger opinion on this.

Reply:
Yeah we might not need this at all. It looks like DAI won't even be using it at this point. Originally we thought they might